WORKSPACE

```diff
@@ -19,3 +23,16 @@ local_repository(
 
 load("@TOOLCHAINS//:toolchains.bzl", "init_toolchains")
 init_toolchains()
+
+##############################################################################
+#Third party repositories
```

Comment:
Can we use http_archive instead of git_repository? It is better and also aligns with the practice from tensorflow

Reply:
will change